src/backend/api/reviews.js

```diff
@@ -0,0 +1,72 @@
+const express = require("express");
+const knex = require("../database");
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+  try {
+    const allReview = await knex("review").select();
+    if (allReview) {
+      res.status(200).json({ data: allReview, message: "ok" });
+    }
+  } catch (error) {
+    console.error(error);
+    res.status(500).send("something wrong");
+  }
+});
+
+router.get("/:id", async (req, res) => {
+  try {
+    const { id: reviewId } = req.params;
+    const reviewOfMeal = await knex("review").where("id", "=", reviewId).select();
+    if (reviewOfMeal) {
+      res.status(400).json({ data: reviewOfMeal, message: "not found" });
+    }
```

Comment:
In the case that a review with a given `id` is not found, no response is sent. This makes Postman just wait endlessly for a response.